tools/test/examples/examples.json

```diff
@@ -339,7 +339,7 @@
         "sub-repo-example": false,
         "subs": [],
         "features" : [],
-        "targets" : ["ARM_MUSCA_A1", "ARM_MUSCA_B1", "K64F", "CY8CKIT_064S2_4343W"],
+        "targets" : ["ARM_MUSCA_B1", "K64F"],
```

Comment:
Could you replace with `CY8CKIT_064B0S2_4343W` please? We should run with a PSoC 64 target. This target will be available after https://github.com/ARMmbed/mbed-os/pull/13243 is merged.